packages/babel-types/src/definitions/experimental.js

```diff
@@ -61,23 +61,3 @@ defineType("ExportNamespaceSpecifier", {
     }
   }
 });
-
-defineType("RestProperty", {
```

Comment:
It if is doable for a type, I'd vote we do it, but it won't work for all of them.

For `.warn` I'm not sure.